src/fileformats/ocd_file_export.cpp

```diff
@@ -22,22 +22,858 @@
+	    {   0,  64,  64 },
+	    {   0,  64, 128 },
+	    {   0,  64, 192 },
+	    {   0,  64, 255 },
+	    {   0, 128,   0 },
+	    {   0, 128,  64 },
+	    {   0, 128, 128 },
+	    {   0, 128, 192 },
+	    {   0, 128, 255 },
+	    {   0, 192,   0 },
+	    {   0, 192,  64 },
+	    {   0, 192, 128 },
+	    {   0, 192, 192 },
+	    {   0, 192, 255 },
+	    {   0, 255,   0 },
+	    {   0, 255,  64 },
+	    {   0, 255, 128 },
+	    {   0, 255, 192 },
+	    {   0, 255, 255 },
+	    {  64,   0,   0 },
+	    {  64,   0,  64 },
+	    {  64,   0, 128 },
+	    {  64,   0, 192 },
+	    {  64,   0, 255 },
+	    {  64,  64,   0 },
+	    {  64,  64,  64 },
+	    {  64,  64, 128 },
+	    {  64,  64, 192 },
+	    {  64,  64, 255 },
+	    {  64, 128,   0 },
+	    {  64, 128,  64 },
+	    {  64, 128, 128 },
+	    {  64, 128, 192 },
+	    {  64, 128, 255 },
+	    {  64, 192,   0 },
+	    {  64, 192,  64 },
+	    {  64, 192, 128 },
+	    {  64, 192, 192 },
+	    {  64, 192, 255 },
+	    {  64, 255,   0 },
+	    {  64, 255,  64 },
+	    {  64, 255, 128 },
+	    {  64, 255, 192 },
+	    {  64, 255, 255 },
+	    { 128,   0,   0 },
+	    { 128,   0,  64 },
+	    { 128,   0, 128 },
+	    { 128,   0, 192 },
+	    { 128,   0, 255 },
+	    { 128,  64,   0 },
+	    { 128,  64,  64 },
+	    { 128,  64, 128 },
+	    { 128,  64, 192 },
+	    { 128,  64, 255 },
+	    { 128, 128,   0 },
+	    { 128, 128,  64 },
+	    { 128, 128, 128 },
+	    { 128, 128, 192 },
+	    { 128, 128, 255 },
+	    { 128, 192,   0 },
+	    { 128, 192,  64 },
+	    { 128, 192, 128 },
+	    { 128, 192, 192 },
+	    { 128, 192, 255 },
+	    { 128, 255,   0 },
+	    { 128, 255,  64 },
+	    { 128, 255, 128 },
+	    { 128, 255, 192 },
+	    { 128, 255, 255 },
+	    { 192,   0,   0 },
+	    { 192,   0,  64 },
+	    { 192,   0, 128 },
+	    { 192,   0, 192 },
+	    { 192,   0, 255 },
+	    { 192,  64,   0 },
+	    { 192,  64,  64 },
+	    { 192,  64, 128 },
+	    { 192,  64, 192 },
+	    { 192,  64, 255 },
+	    { 192, 128,   0 },
+	    { 192, 128,  64 },
+	    { 192, 128, 128 },
+	    { 192, 128, 192 },
+	    { 192, 128, 255 },
+	    { 192, 192,   0 },
+	    { 192, 192,  64 },
+	    { 192, 192, 128 },
+	    { 192, 192, 192 },
+	    { 192, 192, 255 },
+	    { 192, 255,   0 },
+	    { 192, 255,  64 },
+	    { 192, 255, 128 },
+	    { 192, 255, 192 },
+	    { 192, 255, 255 },
+	    { 255,   0,   0 },
+	    { 255,   0,  64 },
+	    { 255,   0, 128 },
+	    { 255,   0, 192 },
+	    { 255,   0, 255 },
+	    { 255,  64,   0 },
+	    { 255,  64,  64 },
+	    { 255,  64, 128 },
+	    { 255,  64, 192 },
+	    { 255,  64, 255 },
+	    { 255, 128,   0 },
+	    { 255, 128,  64 },
+	    { 255, 128, 128 },
+	    { 255, 128, 192 },
+	    { 255, 128, 255 },
+	    { 255, 192,   0 },
+	    { 255, 192,  64 },
+	    { 255, 192, 128 },
+	    { 255, 192, 192 },
+	    { 255, 192, 255 },
+	    { 255, 255,   0 },
+	    { 255, 255,  64 },
+	    { 255, 255, 128 },
+	    { 255, 255, 192 },
+	    { 255, 255, 255 },
+	};
+	
+#if 0
```

Comment:
It tells you how the magical table above was created (how it can be changed).

I'm fine with moving it out once the table is accepted. However, I will have to invent a way to organize are generators and tools (`SymbolSetTool`, which is a useful test and a generator at the same time.)

Reply:
Understood.